examples/standalone-playground/package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "yarn concurrently 'yarn run -T watch --filter=standalone-playground' 'sleep 10 && npx http-server .'"
   },
   "dependencies": {
-    "@segment/analytics-next": "workspace:^"
+    "@segment/analytics-next": "1.41.1"
```

Comment:
How come these were changed from `workspace`? Should they be pinned to the major version? `^1.41.1`?

Reply:
Yes, they should be pinned! The reason is to a fix yarn constraints issue where all the packages should depend on the same version, but in my research it's much more common to use a pinned version (see storybook monorepo, etc) -- don't worry, development is unaffected -- it still hardlinks to the workspace version!